README.md

```diff
@@ -173,6 +173,12 @@ This project is build with:
 - Twitter: [@Evans_22J](https://twitter.com/Evans_22J)
 - LinkedIn: [@Evans Sitibekiso](https://www.linkedin.com/in/evans-sitibekiso/)
 
+👤 **Edward Odhiambo**
+
+- GitHub: [@whit3d](https://github.com/white3d)
+- Twitter: [@odhiambo_ed](https://twitter.com/odhiambo_ed)
+- LinkedIn: [Edward Odhiambo](https://www.linkedin.com/in/edward-odhiambo-6a462a21b/)
+
 ## 🤝 Contributor
 
```

Comment:
- [ ] It would be better if we write featured test for our User post index page as show in the screenshot below

![Screenshot from 2022-08-04 17-03-40](https://user-images.githubusercontent.com/46388113/182909501-0b2fa7d7-c754-48dd-9141-3b65d795fb83.png)

Reply:
Hello @addod19 ,
I have include all the test state in the picture above. See this please [code](https://github.com/evans22j/Blog-App/blob/275824e473eae52f9a709881648fe222127c51e2/spec/integration/posts_index_spec.rb#L25-L75)